File: MovieTicketBooking/Movie Booking C#/Model/Screen.cs
namespace MovieTicketBooking.Model
{
    public class Screen
    {
        string Id { get; }
        public List<Seat> Seats { get; } = new List<Seat>();
        Theatre Theatre;
        public Screen(string id, Theatre theatre)
        {
            this.Id = id;
            this.Theatre = theatre;
        }
        public Screen(string id, List<Seat> seats, Theatre theatre)
        {
            this.Id = id;
            this.Seats = seats;
            this.Theatre = theatre;
        }
        public void AddSeat(Seat seat)
        {
            this.Seats.Add(seat);
        }
    }
}

File: MovieTicketBooking/Movie Booking Java/src/main/java/org/example/model/Screen.java
package org.example.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NonNull;

import java.util.ArrayList;
import java.util.List;

@Getter
@AllArgsConstructor
public class Screen {

    private final String id;
    private final List<Seat> seats;
    private final Theater theater;

    public Screen(String id, Theater theater) {

        this.id = id;
        this.theater = theater;
        this.seats = new ArrayList<>();
    }

    public void addSeat(@NonNull Seat seat) {
        seats.add(seat);
    }
}
